src/webots/gui/WbMainWindow.cpp

```diff
@@ -908,23 +908,63 @@ QMenu *WbMainWindow::createHelpMenu() {
 
   menu->addSeparator();
 
+  action = new QAction(this);
+  action->setText(tr("&GitHub repository..."));
+  action->setStatusTip(tr("Open the Webots git repository on GitHub."));
+  connect(action, &QAction::triggered, this, &WbMainWindow::openGithubRepository);
+  menu->addAction(action);
+
+  action = new QAction(this);
+  action->setText(tr("Cyberbotics &Website..."));
+  action->setStatusTip(tr("Open the Cyberbotics website."));
+  connect(action, &QAction::triggered, this, &WbMainWindow::openCyberboticsWebsite);
+  menu->addAction(action);
+
   action = new QAction(this);
   action->setText(tr("&Bug Report..."));
-  action->setStatusTip(tr("Report a bug to Cyberbotics."));
+  action->setStatusTip(tr("Report a bug to the GitHub repository."));
   connect(action, &QAction::triggered, this, &WbMainWindow::openBugReport);
   menu->addAction(action);
 
   action = new QAction(this);
-  action->setText(tr("&Support Ticket..."));
-  action->setStatusTip(tr("Open a Support Ticket with Cyberbotics."));
+  action->setText(tr("&Support Ticket (Premier Service)..."));
+  action->setStatusTip(tr("Open a Support Ticket with Cyberbotics. This requires a subscription to the Premier Service"));
   connect(action, &QAction::triggered, this, &WbMainWindow::openSupportTicket);
   menu->addAction(action);
 
+  QMenu *followUsMenu = new QMenu(tr("&Keep informed"), this);
+
   action = new QAction(this);
-  action->setText(tr("Cyberbotics &Website..."));
-  action->setStatusTip(tr("Open the Cyberbotics website."));
-  connect(action, &QAction::triggered, this, &WbMainWindow::showCyberboticsWebsite);
-  menu->addAction(action);
+  action->setText(tr("Subscribe to the &Newsletter..."));
+  action->setStatusTip(tr("Keep informed about the last news about Webots following the Webots newsletter."));
+  connect(action, &QAction::triggered, this, &WbMainWindow::openNewsletterSubscription);
+  followUsMenu->addAction(action);
+
+  action = new QAction(this);
+  action->setText(tr("Join the &Discord chanel..."));
```

Comment:
```suggestion
  action->setText(tr("Join the &Discord channel..."));
```
[chanel](https://www.chanel.com) is something different 😄